README.md

```diff
@@ -56,3 +57,4 @@ object CheckCommand "nextcloud" {
 * 2023-04-07: add token athentification (beccon4)
 ## Authors
 * [Kevin Köllmann](https://github.com/koelle25)
+* [Conrad Beckert](https://github.com/beccon4)
```

Comment:
I would appreciate a new section for contributors, like so:

```
## Contributors
* [Conrad Beckert](https://github.com/beccon4)
```

Reply:
ok